torchtext/experimental/modules/transformer.py

```diff
@@ -0,0 +1,93 @@
+import torch.nn as nn
+import torch.nn.functional as F
+from torch.nn import Linear, Dropout, LayerNorm, ModuleList
+from torchtext.nn import MultiheadAttentionContainer, InProjContainer, ScaledDotProduct
+import copy
+from .embedding import PositionalEmbedding
+
+
+class TransformerEncoderLayer(nn.Module):
```

Comment:
Is there a reason we can't use [torch.nn.TransformerEncoderLayer](https://pytorch.org/docs/master/generated/torch.nn.TransformerEncoderLayer.html#torch.nn.TransformerEncoderLayer) here?

Reply:
We could use it for sure.